Manifest.cs

```diff
@@ -0,0 +1,24 @@
+using OrchardCore.Modules.Manifest;
+using static Lombiq.UIKit.FeatureIds;
+
+[assembly: Module(
+    Name = "Lombiq UIKit",
+    Author = "Lombiq Technologies",
+    Version = "1.0",
+    Description = "Module for reusable shapes containing text editors, dropdown editors, and potentially more complex editors.",
+    Website = "https://github.com/Lombiq/Orchard-UIKit"
+)]
+
+[assembly: Feature(
+    Id = Base,
+    Name = "Lombiq UIKit",
+    Category = "Development",
+    Description = "Module for reusable shapes containing text editors, dropdown editors, and potentially more complex editors."
+)]
+
+[assembly: Feature(
+    Id = ShowCase,
+    Name = "Lombiq UIKit - Showcase page",
+    Category = "Development",
+    Description = "You can reach the showcase page at /UIKitShowCase"
```

Comment:
Ezt kértem a Readme-nél is, itt ugyanúgy helytelen:

```suggestion
    Description = "You can reach the showcase page under /UIKitShowCase."
```